project/project.ts

```diff
@@ -70,6 +65,8 @@ export class TsGdProjectClass {
 
   args: ParsedArgs
 
+  dynamicDefBuilder = new DefinitionBuilder(this)
```

Comment:
nit: naming - perhaps `tsDefinitionBuilder` (and `TsDefinitionBuilder`)... or just `definitionBuilder`. Since everything in coding is dynamic anyway :)

Reply:
Changed it to definitionBuilder.